keps/sig-cluster-lifecycle/kubeadm/0023-kubeadm-config.md

```diff
@@ -92,20 +91,23 @@ a more sustainable foundation for the evolution of kubeadm itself.
   by the current command but not persisted).
 
 - Improve the current kubeadm config file format by using specialized substructures instead
-  of the current "single flat object with only fields".
+  of the old "single flat object with only fields".
 
 ### Non-Goals
 
 - To steer/coordinate all the implementation efforts for adoption of ComponentConfig across all
   the different Kubernetes components.
 
-- To define a new home for the Bootstrap Token Go structs
+- To define a new home for the Bootstrap Token Go structs.
+
+- To address all possible proposed changes, that were proposed to the kubeadm config format.
+  Specifically we are trying to avoid changes, that have a fairly limited use cases.
 
 ## Proposal
 
 ### Decoupling the kubeadm types from other ComponentConfig types
 
-The `v1alpha2` kubeadm config types currently embeds the ComponentConfig for
+The `v1alpha2` kubeadm config types embeds the ComponentConfig for
```

Comment:
Maybe we should move around the subheadings so they are grouped by version 

e.g. 
- v1beta1 
    - Decoupling the kubeadm types from other ComponentConfig
- v1beta2 
    - (Details ...) 

This way as you read it you get the order of when the changes happened.